docs/docs/testing.md

```diff
@@ -1744,6 +1744,225 @@ Luckily, RedwoodJS has several api testing utilities to make [testing functions
 ## Testing GraphQL Directives
 
 Please refer to the [Directives documentation](./directives.md) for details on how to write Redwood [Validator](./directives.md#writing-validator-tests) or [Transformer](./directives.md#writing-transformer-tests) Directives tests.
+
+
+## Testing Caching
+If you're using Redwood's [caching](services#caching) - we provide a handful of utilities, and patterns, to help you test this too!
+
+Let's say you have a service where you cache the result of products, and individual products
+
+```ts
+export const listProducts: QueryResolvers['listProducts'] = () => {
+  // highlight-next-line
+  return cacheFindMany('products-list', db.product, {
+    expires: 3600,
+  })
+}
+
+export const product: QueryResolvers['product'] = async ({ id }) => {
+  // highlight-next-line
+  return cache(
+    `cached-product-${id}-`,
```

Comment:
```suggestion
    `cached-product-${id}`,
```

Trailing dash there, but then later when you test for the key in `toHaveCached()` there's no trailing dash!